examples/simple_dashboard/Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+hikari = "*"
+rillrate = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.9"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Could we use a requirements.txt file for this as well as keep consistent with the other examples (having a little shell script to run them)

Reply:
fixed on #dec19deb